docs/sources/addons.md

```diff
@@ -30,6 +30,7 @@ All optional add-ons for openHAB 2 are [available in a separate download](https:
 
 | Add-on | Type |
 |--------|------|
+| Denon | Binding |
 | EDS OWServer | Binding |
 | Energenie | Binding |
 | Enocean | Binding |
```

Comment:
Hi...
I am not able to Transmit/Send the Enocean RPS command through OpenHAB whenever I press the Enocean Switch. At this forum also I have seen lot people facing this Issue. Enocean Actor/Actuator just doesn't respond to OpenHAB. May I know, what's the Issue? Awaiting Your Reply... Thanks!